java/Graql.java

```diff
@@ -81,6 +81,10 @@ public static BoundVariable parseVariable(String variable) {
         return parser.parseVariableEOF(variable);
     }
 
+    public static String parseLabel(String label) {
+        return parser.parseLabelEOF(label);
```

Comment:
We should check here if the label contains whitespace or comments, and throw an exception if so.